jerry-core/profiles/README.md

```diff
@@ -108,8 +100,6 @@ defined to `1`.
 
 #### ES6 (ES 2015) features
 
-* `JERRY_ES2015_ARROW_FUNCTION`:
-  Enables or disables the [arrow functions](http://www.ecma-international.org/ecma-262/6.0/#sec-arrow-function-definitions).
 * `JERRY_ES2015_BUILTIN`:
   Enables or disables the built-in updates of the 5.1 standard. There are some differences in those built-ins which available in both [5.1](http://www.ecma-international.org/ecma-262/5.1/) and [2015](http://www.ecma-international.org/ecma-262/6.0/) versions of the standard. JerryScript uses the latest definition by default.
```

Comment:
It would be good here to list the available features when this macro is enabled (just copy the deleted lines from this file).

Reply:
You can get this list [here](https://github.com/jerryscript-project/jerryscript/pull/3240/files#diff-52116e59bc8cbc847a29fc0adf4a41f3R120).

Reply:
Great! I didn't notice it.